src/containers/InteractiveMessage/InteractiveOptions/InteractiveOptions.tsx

```diff
@@ -62,6 +62,7 @@ export const InteractiveOptions: React.SFC<InteractiveOptionsProps> = ({
     if (templateType === LIST) {
       template = (
         <ListReplyTemplate
+          key={index}
```

Comment:
Is it okay to use the index as a key? or should we go for just a random string such as the current date? Sometime back when I used index as key, deepscan gave me an error :(

Reply:
Nope, we should not be directly using the index of the map for the key. I have updated the code.